cargo-cyclonedx/src/generator.rs

```diff
@@ -83,9 +94,9 @@ impl SbomGenerator {
                     top_level_dependencies(&members, &package_ids)?
                 };
 
-            let mut bom: Bom = dependencies.iter().collect();
+            let bom = create_bom(member, dependencies);
 
-            bom.metadata = get_metadata(member.manifest_path());
+            log::debug!("Bom validation: {:?}", &bom.validate());
```

Comment:
Should we drop this line before merging?

Reply:
While we in theory should be constructing a valid BOM, I like being able to prove that with the library. This did expose some problems when initially doing the integration so it's very helpful as a developer.

I think it makes sense to leave this since it's behind a log level switch. We could move it to `log::trace` but I think debug makes more sense because trace is really noisy.